hydra-node/src/Hydra/Chain/ScriptRegistry.hs

```diff
@@ -103,10 +100,9 @@ publishHydraScripts networkId socketPath sk = do
   vk = getVerificationKey sk
 
 -- | Exception raised when building the script publishing transactions.
-data PublishScriptException
-  = FailedToBuildPublishingTx (TxBodyErrorAutoBalance Era)
-  | FailedToFindUTxOToCoverDeposit {totalDeposit :: Coin}
-  deriving (Show)
+newtype PublishScriptException
+  = PublishScriptException (TxBodyErrorAutoBalance Era)
```

Comment:
The data constructor name from before was better (it pointed at a construction error)